nix/pkgs/images/default.nix

```diff
@@ -122,20 +122,7 @@ rec {
     maxLayers = 42;
   };
 
-  services-kiiss-image = dockerTools.buildLayeredImage (servicesImageProps // {
-    name = "mayadata/services-kiiss";
-    contents = [ busybox mayastor ];
-    config = { Entrypoint = [ "/bin/kiiss" ]; };
-    maxLayers = 42;
-  });
-
-  services-kiiss-dev-image = dockerTools.buildImage (servicesImageProps // {
-    name = "mayadata/services-kiiss-dev";
-    contents = [ busybox mayastor ];
-    config = { Entrypoint = [ "/bin/kiiss" ]; };
-  });
-
-  mayastor-client-image = dockerTools.buildImage (servicesImageProps // {
+  mayastor-client-image = dockerTools.buildImage (clientImageProps // {
```

Comment:
Do you think we could stop repeating ourselves? So it's not `images.mayastor-images`, it's `images.mayastor`?

Reply:
I think that's a good suggestion and what I was trying to do when building the services. I can make that change.